docs/_includes/counter.adoc

```diff
@@ -1,9 +1,66 @@
 ////
 Included in:
 
-- user-manual: Counter Attributes
+- user-manual: Counters
 ////
 
+// See issue #550, #561, https://github.com/asciidoctor/asciidoctor/issues/1559
+// Focus on simple use case.
+// {counter:name:start_value} is only for first use.
+// document attributes and counters are NOT the same thing, but modifying a document attribute with the same name as the counter modifies the counter at the same time.
+
+Counters are used to generate sequences of numbers or latin characters.
+Possible uses are to create inline itemised lists, paragraph numbers, or serial numbers.
+
+Counters are created and used by prefixing the counter name with `counter:` and surrounding it with curly brackets.
```

Comment:
I feel like a parallel to an attribute references is missing here. I'd like it to say:

Counters are a special type of document attribute.
Counters are declared and displayed using a modified attribute reference in which the attribute name is prefixed with `counter:`.
Since they are attributes, counter names follow the same rules as attribute names.

(Move discussion of the start value to the next paragraph).